pkgs/servers/nextcloud/default.nix

```diff
@@ -32,6 +33,10 @@ let
         inherit packages;
       };
 
+      patchPhase = ''
```

Comment:
`postPatch`, please.

Overriding `patchPhase` means that `patches = []` is not applied anymore. We don't have any in here, but it's not too uncommon for people to do `nextcloud.overrideAttrs (_: { patches = [ custom-patches ]; })`.